index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>My portafolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="css/style.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+
+<body>
+  <nav class="toolbar">
+    <p id="toolbar-name">Richard Burneo</p>
+    <i id="hamburguer" class="fa fa-bars" aria-hidden="true"></i>
+  </nav>
+  <section class="headline">
+    <h1>Hey There I'm Richard</h1>
+    <h3 id="h3-headline">I am Software Developer</h3>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+      what
+      you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="social-network">
+      <i class="fa-brands fa-github social"></i>
+      <i class="fab fa-linkedin-in social"></i>
+      <i class="fa-brands fa-angellist social"></i>
+      <i class="fa-brands fa-twitter social"></i>
+      <i class="fa-solid fa-m social"></i>
+    </div>
```

Comment:
- [ ] For this section, kindly use a `ul` tag to group the social links,  an `li` tag per icon, and finally an `a` tag for the icons.
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.